docs/src/intro/overview.md

```diff
@@ -41,8 +41,8 @@ For reasons explained [here](../design/stack/main.md), the VM imposes the restri
 The *advice provider* component is responsible for supplying nondeterministic inputs to the VM. These inputs only need to be known to the prover (i.e., they do not need to be shared with the verifier).
 
 The advice provider consists of three components:
-* **Advice tape** which is a one-dimensional array of values. The VM can access only the head of the tape. That is the VM can either remove values from the head of the tape or inject new values at the head of the tape. Formally, this means that the advice tape is actually a stack.
-* **Advice map** which is a key-value map where keys are words and values are vectors of field elements. The VM can copy values from the advice map onto the advice tape as well as insert new values into the advice map (e.g., from a region of memory).
+* **Advice stack** which is a one-dimensional array of values. Being a stack, the VM can access only its head. That is the VM can either remove values from the head, or inject new values at the head.
```

Comment:
I'd change this to:

**Advice stack** which is a one-dimensional array of field elements. Being a stack, the VM can either push new elements on to the advice stack, or pop the elements from the top of the stack.

Reply:
Done!